src/app/[countryCode]/(main)/order/[id]/transfer/[token]/page.tsx

```diff
@@ -0,0 +1,38 @@
+import { Heading, Text } from "@medusajs/ui"
+import TransferActions from "@modules/order/components/transfer-actions"
+import TransferImage from "@modules/order/components/transfer-image"
+
+export default async function TransferPage({
+  params,
+}: {
+  params: { id: string; token: string }
+}) {
+  const { id, token } = params
+
+  return (
+    <div className="flex flex-col gap-y-4 items-start w-3/5 mx-auto mt-10 mb-20">
```

Comment:
suggestion: would go for "2/5" here so it's easier to read, also this section is wider than the header container on some breakpoints